release-notes/8.0/preview/api-diff/preview7/Microsoft.NETCore.App/8.0-preview7_System.Diagnostics.Metrics.md

```diff
@@ -0,0 +1,13 @@
+# System.Diagnostics.Metrics
+
+``` diff
+ namespace System.Diagnostics.Metrics {
++    public interface IMeterFactory : IDisposable {
++        Meter Create(MeterOptions options);
++    }
++    public static class MeterFactoryExtensions {
++        public static Meter Create(this IMeterFactory meterFactory, string name, string? version = null, IEnumerable<KeyValuePair<string, object?>>? tags = null);
++    }
+ }
```

Comment:
I think these moved interfaces from one assembly to other assembly. but existed in preview 5 as I recall.

Reply:
Was that movement expected, or are you saying the API diff tool has a bug?

Reply:
The move is expected. If we are documenting the API diff difference should be noted as moved and not introduced APIs.